### ISSUE TITLE
Build failure: termusic

### ISSUE THREAD

**Issue by Elias-Ainsworth** [NixOS/nixpkgs#339280]
### Steps To Reproduce

1. build termusic (I ran `nix-shell -p termusic`)

### Build log

https://gist.github.com/Elias-Ainsworth/95e49d91dcc0a4dd93a7ff4f83c6ef5a

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.47-xanmod1, NixOS, 24.11 (Vicuna), 24.11.20240824.d0e1602`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - nixpkgs: `/nix/store/ia1zpg1s63v6b3vin3n7bxxjgcs51s2r-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by eclairevoyant** [comment]
another case of #332957

**Comment by eclairevoyant** [comment]
actually already fixed in #334860
